### DISCUSSION TITLE
Bookkeeper still work when non-faulty bookies less than Ensemble

### DISCUSSION THREAD

**Discussion by SpicyChickenFLY** [apache/pulsar#20578] [Q&A]
I deployed a pulsar cluster with 6 bookies on different machines and set persistent policies like E=7 W=7 A=7. However, the bookkeeper still works and Pulsar client successfully produces messages. Why would this situation happen?

BTW, in which case below will bookkeeper become not available?
1) non-faulty bookies less than Ensemble
2) non-faulty bookies less than Write Quorum
3) faulty bookies great than Ack Quorum

**Comment by tisonkun** [comment]
cc @eolivelli @hangc0276

**Reply by SpicyChickenFLY** [reply]
Hi, Is there any new progress on this discussion?

**Reply by hangc0276** [reply]
It is not the expected behavior. Would you please share the broker's logs and check the ledger's metadata?
